scripts/setup-master.sh

```diff
@@ -30,8 +30,14 @@ echo "Adding replication specific configuration"
   echo "wal_level = replica"
   echo "archive_mode = on"
   echo "archive_command = '${ARCHIVE_COMMAND}'"
-  echo "max_wal_senders = 5"
-  echo "wal_keep_size = 512"
+  echo "max_wal_senders = 10"
+  echo "wal_keep_size = 2048"
   echo "hot_standby = on"
-  echo "synchronous_standby_names = '*'"
 } >>"$PGDATA"/postgresql.conf
+
+if [[ ${SYNC_REPLICATION^^} == TRUE ]]; then
+  echo "Adding synchronous replication specific configuration"
+  {
+    echo "synchronous_standby_names = '*'"
```

Comment:
There's no async equivalent of this?

Reply:
Nope. Replication is asynchronous by default so there is really no need to be explicit about it.
https://www.postgresql.org/docs/13/warm-standby.html#SYNCHRONOUS-REPLICATION

Reply:
So I assume that it's just a fire and forget, there's no ACL?

Reply:
It's just that.